zhaquirks/tuya/ts0601.py

```diff
@@ -230,8 +228,6 @@ class TuyaMoesCover0601(TuyaWindowCover):
             ("_TZE200_zah67ekd", "TS0601"),
             ("_TZE200_xuzcvlku", "TS0601"),
             ("_TZE200_rddyvrci", "TS0601"),
-            ("_TZE200_nueqqe6k", "TS0601"),
-            ("_TZE200_gubdgai2", "TS0601"),
```

Comment:
Way deleting this 2 devices ?

Reply:
my bad , i use a previous reference file ,  those two must be keeped

Reply:
it's fixed